src/main/diagnostics/rdBinInShell.ts

```diff
@@ -71,6 +73,12 @@ class RDBinInShellPath implements DiagnosticsChecker {
       fixes,
     };
   }
+
+  static removeTrailingSlash(s: string): string {
+    const m = /(.*?)(\/+)$/.exec(s);
+
+    return m ? (m[1] || '/') : s;
```

Comment:
Simler expression:

```suggestion
    return s.replace(/(.)\/*$/, "$1");
```

Demo:

```
> "/foo/bar//".replace(/(.)\/*$/, "$1")
'/foo/bar'
> "/".replace(/(.)\/*$/, "$1")
'/'
> "//".replace(/(.)\/*$/, "$1")
'/'
> "".replace(/(.)\/*$/, "$1")
''
```